Sources/Valet/Internal/Keychain.swift

```diff
@@ -109,6 +109,10 @@ internal final class Keychain {
         var secItemQuery = options
         secItemQuery[kSecAttrAccount as String] = key
         
+        if #available(iOS 13.0, tvOS 13.0, watchOS 6.0, macOS 10.15, *) {
+            secItemQuery["nleg"] = true // kSecUseDataProtectionKeychain for Xcode 9 and Xcode 10 compatibility.
```

Comment:
Is the `#available` check here not enough to use `kSecUseDataProtectionKeychain`?  If not, should we move this to a `Constants` enum or something along those lines?

Same applies to `SecItem.swift`

Reply:
It isn't unfortunately, since this branch still needs to support Xcode 10 and Xcode 9. Neither of those Xcode's know about this field, and the `if #available` check isn't like an `#if` in Objective-C – the code inside still gets compiled if the condition isn't met.

This kinda code sucks, and is one of the things that makes writing Swift code that supports multiple Xcode versions hard. It's one of the reasons I'm really looking forward to the develop--4.0 branch, since I'll be able to use the proper constant here.

I did think about making a local constant for now, but given that I'm going to bring this fix to develop--4.0 with the actual constants as soon as this lands... it didn't seem worth the time 😅.